docs/changelog.rst

```diff
@@ -1,6 +1,37 @@
 Changes in IPython kernel
 =========================
 
+4.6
+---
+
+4.6.0
+*****
+
+`4.6.0 on GitHub <https://github.com/ipython/ipykernel/milestones/4.6.0>`__
+
+- Add to API `DisplayPublisher.publish` two new fully backward-compatible
+  keyword-args:
+    - `update: bool`
+    - `transient: dict`
+- Add  a new dict, `transient`, to message spec for `publish`. For a display
```

Comment:
I'd change this to

> support new `transient` key in display_data messages...

ipykernel doesn't define these things, the msg_spec in `jupyter_client` does.